rastervision/registry.py

```diff
@@ -153,8 +153,9 @@ def __init__(self):
         }
 
         self.experiment_runners = {
-            rv.LOCAL: rv.runner.LocalExperimentRunner,
-            rv.AWS_BATCH: rv.runner.AwsBatchExperimentRunner
+            rv.OLD_LOCAL: rv.runner.LocalExperimentRunner,
+            rv.AWS_BATCH: rv.runner.AwsBatchExperimentRunner,
+            rv.LOCAL: rv.runner.NewLocalExperimentRunner
```

Comment:
I think it would be clearer if `rv.OLD_LOCAL` mapped to `rv.runner.OldLocalExperimentRunner` and `rv.LOCAL` mapped to `rv.runner.LocalExperimentRunner`

Reply:
Done.